lib/menu.rb

```diff
@@ -0,0 +1,20 @@
+class Menu
+  attr_reader :list
+
+  def initialize
+    @list = VEGETARIAN_MENU
+  end
+
+  def print_menu
+    list.each do | dish, price |
+      dish_list = (dish.to_s).ljust(25)
+      price_list = "£#{price.to_s}".rjust(25)
```

Comment:
Could you make an extra method that can generate a single line of the menu to make this code clearer?